orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/OrcaMessageHandler.kt

```diff
@@ -52,6 +58,7 @@ internal interface OrcaMessageHandler<M : Message> : MessageHandler<M> {
         .taskById(taskId)
         .let { task ->
           if (task == null) {
+            log.error("InvalidTaskId: Unable to find task {} in existing tasks {} while processing message {}", taskId, stage.tasks.map { it.id }, this.javaClass)
```

Comment:
you should probably switch this back to printing `this` instead of `this.javaClass`. While I have a hard time believing the message can be corrupt, it would be useful to ascertain whether it is or is not